Replicator/Replicator.hh

```diff
@@ -119,27 +119,15 @@ namespace litecore::repl {
 
         // Replicator owns multiple subRepls, so it doesn't use _collectionIndex, and therefore we must
         // pass the collectionIndex manually for log calls
-        template <class... Args>
-        inline void cLogInfo(CollectionIndex idx, const char* fmt, Args... args) const {
-            const char* fmt_ = formatWithCollection(fmt);
-            Logging::logInfo(fmt_, idx, args...);
-        }
-
-        template <class... Args>
-        inline void cLogVerbose(CollectionIndex idx, const char* fmt, Args... args) const {
-            const char* fmt_ = formatWithCollection(fmt);
-            Logging::logVerbose(fmt_, idx, args...);
-        }
+#define cWarn(IDX, FMT, ...)       _logAt(Warning, "Coll=%i " FMT, (IDX), ##__VA_ARGS__)
```

Comment:
Hmmm, if these are not used outside of Replicator.cc then it would be better to define them there rather than having them hanging around globally for anything that includes Replicator.hh

Reply:
True. I'll move them to Replicator.cc